tests/spread/integration/pwgen/task.yaml

```
@@ -0,0 +1,11 @@
+summary: Integration tests for pwgen
+
+execute: |
+  rootfs="$(install-slices pwgen_bins)"
```

```suggestion
  rootfs="$(install-slices base-files_base pwgen_bins)"
```

I suspect the spread tests are failing because of this. Maybe `libc6_libs` should come up with `base-files_base` or `base-files_lib`. Anyway, until that happens, let's please install `base-files_base` too.